Creating a pull request for the heatmap of the smashing magazine web page.

len23 opened the pull request:
- Created html with the all semantic information.
- Created CSS with all the recomendations.
- The project is validated.

GabrielJSuarez committed:
Created skeleton files

- created index file
- created style file
- create assets folder

len23 committed:
Test push

len23 committed:
Add navBar

GabrielJSuarez committed:
Added reset library

- linked normalize css
- linkd style css
- added linters folder

GabrielJSuarez committed:
Update style.css

added globals

len23 committed:
Add styles to navbar

GabrielJSuarez committed:
Style.css bugfixing

- styled the navbar

GabrielJSuarez committed:
Update index.html

Added semantic html and divs to style the front page

GabrielJSuarez committed:
Created the grid

- Created html and css for the grid container and styles

GabrielJSuarez committed:
Starting to style the fonts

- added styles to the fonts of the document

GabrielJSuarez committed:
Update style.css

Fixed centering of the article headers

len23 committed:
Add spans and font-families

len23 committed:
Add comment text and the photo author

len23 committed:
Fix margin for h2

GabrielJSuarez committed:
Changed HTML/CSS

- Styled imags and headings

GabrielJSuarez committed:
Changes in HTML/CSS

- Added structure to style the hash symbols
- Added styles to the HTML

len23 committed:
Add posts mapheat section

GabrielJSuarez committed:
Uptaded html/css

- added tags and style for the membership section heatmap

GabrielJSuarez committed:
Changes in html/css

- Added tags and styles for the episodes section

len23 committed:
Add person heat section

GabrielJSuarez committed:
Changes in HTML/CSS

- finished Guides and design section's heatmap

GabrielJSuarez committed:
Styling the footer

- Aded html skelon for the footer's heatmap

len23 committed:
Add footer styles heatmap and fix styles

GabrielJSuarez committed:
Linter bugfixing

Bugfixing linter alerts

GabrielJSuarez committed:
Update style.css

Bugfixing

GabrielJSuarez committed:
Create screenshot.png

added screenshot of the page

GabrielJSuarez committed:
Update README.md

MuriloRoque commented on style.css:
```diff
@@ -0,0 +1,481 @@
+    color: #fff;
+    width: 75%;
+    height: 35rem;
+    margin-top: 3rem;
```
You are using this property 9 times throughout your CSS, maybe it would be better to make an `mt-3` class? :wink:

GabrielJSuarez replied:
FIxed

MuriloRoque commented on style.css:
```diff
@@ -0,0 +1,481 @@
+
+body {
+    max-width: 144rem;
+    margin: 0 auto;
```
You are using this property 6 times throughout your CSS, maybe it would be better to make an `m-0` class? :wink:

GabrielJSuarez replied:
fixed

MuriloRoque reviewed (changes requested):
## Status: Changes requested :red_circle: 

### Hello @len23 and @GabrielJSuarez :wave: 

### You have done an amazing job in this project, your page is really looking good, congrats :clap: 

### But before we move on, we need to fix a few issues, please check my comments below.

## Page Structure

- [ ] Make sure to build your layout using a width of **1440px** on the original page. You can use the Developer Tools to match that screen size.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/pageLayoutScreenSize.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/))

*Note:* Most of the issues in your page are there because of the above requirement, check my screenshots of the original 1440px page.

- [ ] In these rows, we can see that the original has 3 rows with 3 columns each. Yours have one row with 2 columns, then one with 3 columns, then another one with 3 columns, then 3 small rows with only one small column:

***YOURS:***

![Screenshot from 2020-08-27 16-54-43](https://user-images.githubusercontent.com/59691171/91488821-43dc6380-e886-11ea-82dd-e5f4f84404ad.png)
<br>

***ORIGINAL:***

![Screenshot from 2020-08-27 16-55-00](https://user-images.githubusercontent.com/59691171/91488858-50f95280-e886-11ea-9628-cf0b28cd80e4.png)
<br>

## General

- [ ] Please check my comments in your code.

## Summary

- Your repository is very professional, keep up the good work :rocket: 
- Your HTML is semantic and you are using CSS best practices overall :+1: 
- Your page layout is nicely implemented, we just need to fix that screen size detail and we can move forward :100: 

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[![Twitter Follow](https://img.shields.io/twitter/follow/MuriloRoquePai1?label=Follow%20Murilo%20on%20Twitter&style=social)](https://twitter.com/MuriloRoquePai1)

GabrielJSuarez committed:
Bugfixes

- Bugfixing to properly show a grid element
- Refactoring CSS code to avoid class repeats

GabrielJSuarez committed:
Merge branch 'cloneBranch-Feature' of https://github.com/GabrielJSuarez/designTeardown-Microverse into cloneBranch-Feature

kevcap reviewed (approved):
# [STATUS] PROJECT APPROVED 🍾

### Congratulations @len23 and @GabrielJSuarez ! Good job making my previously requested changes. You guys did an excellent job, since you guys meet all the requirements requested by our great TSE @MuriloRoque , you have my approval to go ahead and merge this pull request!

_**Clean and modular code, following the best practices and the design a looks great, keep it up!**_ 🥇

<img src="https://thumbs.gfycat.com/WholeDevotedDiplodocus-size_restricted.gif">

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> <img  alt="gmail"  src="https://img.shields.io/badge/gmail-%20- green?logo=gmail&style=social"></a> [![Twitter Follow](https://img.shields.io/twitter/follow/kevinnnmn?style=social)](https://twitter.com/intent/follow?screen_name=kevinnnmn) [![](https://img.shields.io/badge/Slack-Kevin%20Alves-green?logo=slack&style=social)](https://microverse-students.slack.com/team/UT86D3AP4)
